cameraserver/src/main/native/cpp/cameraserver/CameraServer.cpp

```diff
@@ -47,8 +48,14 @@ struct CameraServer::Impl {
 };
 
 CameraServer* CameraServer::GetInstance() {
-  static CameraServer instance;
-  return &instance;
+  struct Creator {
+    static void* call() { return new CameraServer{}; }
+  };
+  struct Deleter {
+    static void call(void* ptr) { delete reinterpret_cast<CameraServer*>(ptr); }
```

Comment:
Does this need a reinterpret_cast?  static_cast from void* should work.

Reply:
Probably. Fixed.